release/action.yaml

```diff
@@ -34,10 +34,13 @@ runs:
   steps:
     - name: Checkout
       uses: actions/checkout@v3
-      if: inputs.checkout-repo
```

Comment:
Is this a mistaken remove? If not the input should be pulled

Reply:
This removed on purpose. See how it is used in the line below. It was a bug from a previous pr

Reply:
Now I see the error. Undid this change and my other stupid change from the previous PR